### PR TITLE
 [Behat] Add selenium for test with JS

### PR DESCRIPTION
**Pull request by blazarecki**

### CODE REVIEW

**Comment by B-Galati** [comment]
Fuck yea :D

File: composer.json [hunk 1]
```diff
@@ -71,7 +71,8 @@
         "symfony/psr-http-message-bridge": "^1.0",
         "myclabs/php-enum": "^1.5",
         "knplabs/knp-snappy-bundle": "^1.5",
-        "h4cc/wkhtmltopdf-amd64": "^0.12.4"
+        "h4cc/wkhtmltopdf-amd64": "^0.12.4",
+        "behat/mink-selenium2-driver": "^1.3"
```
Comment (B-Galati) [inline composer.json]: should in require-dev

File: circle.yml [hunk 2]
```diff
@@ -39,6 +40,9 @@ dependencies:
         - openssl genrsa -out var/private.key 1024
         - openssl rsa -in var/private.key -pubout -out var/public.key
         - chmod 660 var/{private,public}.key
+        - wget http://selenium-release.storage.googleapis.com/2.53/selenium-server-standalone-2.53.1.jar
```
Comment (B-Galati) [inline circle.yml]: Would be better through HTTPS

File: features/bootstrap/SecurityContext.php [hunk 3]
```diff
@@ -43,17 +43,24 @@ private function logAs(UserInterface $user): void
     {
         $driver = $this->getSession()->getDriver();
         $session = $this->getContainer()->get('session');
+
+        if ($driver instanceof Selenium2Driver) {
+            $page = $this->getSession()->getPage();
+
+            $this->visitPath('/connexion');
+            $page->findField('_adherent_email')->setValue($user->getUsername());
+            $page->findField('_adherent_password')->setValue(\AppBundle\DataFixtures\ORM\LoadAdherentData::DEFAULT_PASSWORD);
```
Comment (Remg) [inline features/bootstrap/SecurityContext.php]: any reason not to declare a `use` statement for `LoadAdherentData` ?

Comment (B-Galati) [reply features/bootstrap/SecurityContext.php]: no

**Pull request merged**